packages/core/src/modules/vc/W3cCredentialService.ts

```diff
@@ -336,23 +336,27 @@ export class W3cCredentialService {
     return w3cCredentialRecord
   }
 
-  public async getAllCredentials(): Promise<W3cVerifiableCredential[]> {
-    const allRecords = await this.w3cCredentialRepository.getAll()
-    return allRecords.map((record) => record.credential)
+  public async removeCredentialRecord(id: string) {
+    const cred = await this.w3cCredentialRepository.getById(id)
+    await this.w3cCredentialRepository.delete(cred)
```

Comment:
nit, but we try to avoid abbreviations as much as possible

```suggestion
    const credential = await this.w3cCredentialRepository.getById(id)
    await this.w3cCredentialRepository.delete(credential)
```